common/src/main/java/com/microsoft/identity/common/internal/util/ProcessUtil.java

```diff
@@ -57,8 +60,7 @@ public static boolean isBrokerProcess(@NonNull final Context context) {
     /**
      * Returns the running process name.
      */
-    @Nullable
-    private static String getProcessName(@NonNull final Context context) {
+    private static @Nullable String getProcessName(@NonNull final Context context) {
```

Comment:
Under what circumstances will this return null?

Reply:
when we can't find a matching process id, which should never happen.

I'm hesitant to throw a RuntimeException here (IllegalStateException?). Maybe that's the right thing to do.

Reply:
Keeping it the same for now.